app/components/layout.js

```diff
@@ -1,16 +1,16 @@
-import React from 'react'
-import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import Navbar from '../components/common/Navbar'
-import Row from './row'
-import {connectToServer, handshake} from '../redux/actions'
-import DisplayHandshake from './displayhandshake'
-import Toolbelt from './toolbelt'
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+// import {bindActionCreators} from 'redux';
+import Navbar from '../components/common/Navbar';
+// import Row from './row';
```

Comment:
I removed this in [another commit](https://github.com/code-mancers/rbkit-electron-redux/pull/5/files#diff-ebda658fb74330dd9e6b1cb58b9ad64fR6) which is #5